src/auto_version/tests/test_autoversion.py

```diff
@@ -129,6 +129,39 @@ def test_custom_field_set(self):
         self.assertEqual(updates["UNRELATED_STRING"], "apple")
 
 
+class TestMultiFileBumps(unittest.TestCase):
+    call = functools.partial(main, config_path="double_target.toml")
+
+    @classmethod
+    def setUpClass(cls):
+        dir = os.path.dirname(__file__)
+        os.chdir(os.path.abspath(dir))
+
+    def tearDown(self):
+        self.call(set_to="19.99.0")
+
+    def test_bump_patch(self):
+        old, new, updates = self.call(bump="patch", release=True)
+        self.assertEqual(
+            updates,
+            {
+                "RELEASE": True,
+                "VERSION": "19.99.1",
+                "VERSION_AGAIN": "19.99.1",
+                "STRICT_VERSION": "19.99.1",
+            },
+        )
+        with open("example2.py", "r") as f:
+            second_file = f.read()
+            self.assertEqual(second_file, '''LOCK = False
+RELEASE = True
```

Comment:
Would I `readlines()`, the assert would look less ugly; but then my python was glueing up two lines together, faithfully keeping `\n` in the middle, and I decided that the life's too short.